proto/nvme_gw.proto

```diff
@@ -32,15 +32,21 @@ service NVMEGateway {
 	// Add NVMf namespace to nvme subsystem
 	rpc nvmf_subsystem_add_ns(subsystem_add_ns_req) returns(nsid) {}
 
+	// Delete namespace from subsystem
+	rpc nvmf_subsystem_remove_ns(ns_delete_req) returns(req_status) {}
+
 	// Add Host to a subsystem (so all namespaces in that subsystem)
 	rpc nvmf_subsystem_add_host(subsystem_add_host_req) returns (req_status) {}
 
-	// Allow any host access to given subsystem. #ifdef TEST
-	rpc nvmf_subsystem_allow_any_host(subsystem_allow_any_host_req) returns (req_status) {}
+	// Delete host from subsystem
+	rpc nvmf_subsystem_remove_host(host_delete_req) returns (req_status) {}
```

Comment:
I just noticed that we sometimes use `delete` and sometimes `remove`. Can we keep this consistent and use `delete` or `remove` everywhere.

Reply:
I've created issue #14 to discuss more about this for a future PR.